### ISSUE TITLE
Noctua annotation missing on snapshot??????

### ISSUE THREAD

**Issue by ukemi** [geneontology/pipeline#330]
I may be missing something really basic, but when I look at the products directory on snapshot I am not seeing the annotation directory that would have noctua annotations in it. Did I miss that they were moving somewhere?

**Comment by kltm** [comment]
@ukemi Just wanted to confirm that you found these and that's why this was closed?

**Comment by ukemi** [comment]
Correct. Although when I first looked, I could have sworn the folder was not showing up in my browser.
